extensions/ql-vscode/src/view/method-modeling/MethodModeling.tsx

```diff
@@ -1,9 +1,35 @@
 import * as React from "react";
+import { styled } from "styled-components";
+import { ModelingStatusIndicator } from "../model-editor/ModelingStatusIndicator";
+
+const Container = styled.div`
+  background-color: var(--vscode-peekViewResult-background);
+  padding: 0.3rem;
+  margin-bottom: 1rem;
+`;
+
+const Title = styled.div`
+  padding-bottom: 0.3rem;
+  font-size: 1.2em;
+`;
+
+const DependencyBox = styled.div`
```

Comment:
(minor) Should we call it `DependencyContainer` to match the other name?

Reply:
yes!

Reply:
Did you mean to rename this?